suite-native/module-device-settings/src/screens/DeviceAuthenticitySuccessScreen.tsx

```diff
@@ -0,0 +1,34 @@
+import { Button, PictogramTitleHeader, VStack } from '@suite-native/atoms';
+import { Translation } from '@suite-native/intl';
+import { Screen, ScreenHeader, useNavigateToInitialScreen } from '@suite-native/navigation';
+
+export const DeviceAuthenticitySuccessScreen = () => {
+    const navigateToInitialScreen = useNavigateToInitialScreen();
+
+    return (
+        <Screen
+            header={<ScreenHeader closeActionType="close" closeAction={navigateToInitialScreen} />}
+        >
+            <VStack flex={1} spacing="sp32">
+                <VStack flex={1} spacing="sp40" justifyContent="center">
+                    <PictogramTitleHeader
+                        variant="success"
```

Comment:
Based on Figma, I believe there's `icon="check"` missing since the previous version of the component (as mentioned by you in Figma) actually had icon `checkCircle`.

Reply:
agreed with @shenkys